CHANGELOG.rst

```diff
@@ -73,6 +73,7 @@ Fixed
 - fixed the hanging HTTP call with ``ner_duckling_http`` pipeline
 - fixed Interactive Learning intent payload messages saving in nlu files
 - fixed DucklingHTTPExtractor dimensions by actually applying to the request
+- ``MultiProjectImporter`` now imports files in the order of the import statements
```

Comment:
we released 1.4 in the meanwhile, so the location in the changelog has changed

Reply:
ok, the changelog description has been moved to unreleased version. 😄